aqua-ethos-setup/aqua_ethos.sh

```diff
@@ -123,6 +151,90 @@ function makePut {
 	fi
 }
 
+function get_gateway_id {
+    GATEWAY=$(nakeGET servers body| jq "[.[].id]")
+}
```

Comment:
The `getGatewayId` function should return the gateway ID via `echo` for consistency